script/lib/filesystem.py

```diff
@@ -51,15 +51,17 @@ def download_and_extract(destination, url):
     with contextlib.closing(urllib2.urlopen(url)) as u:
       total = int(u.headers['content-length'])
       done = 0
+      last_length = 0
       while True:
         chunk = u.read(1024)
```

Comment:
@MarshallOfSound I think this change introduced in  #584 is making our downloads really slow on our Mac CI.  Before #584 this was  `chunk = u.read(1024*1024)`, so now we are effectively reading smaller chunks and the effect is that downloads of libcc are taking over an hour our nightly builds, eg see: https://github.visualstudio.com/electron/_build/results?buildId=2522&view=logs (bootstrap times out job at an hour) vs https://github.visualstudio.com/electron/_build/results?buildId=2169 where bootstrap took 9 minutes

Reply:
Hm, I did that to cosmetically improve the download number incrementing.  I guess if writing to stdout is slow on CI it could have a very noticeable impact.  Tbh if this was written in node we could have streams and do this a lot nicer.  For now let's just bump it back up to 1024*1024

Reply:
I'm testing it now with 1mb (1024*1024) vs 1k.

Reply:
@MarshallOfSound yeah... its much faster now!
This was with 1024: https://github.visualstudio.com/electron/_build/results?buildId=2380&view=logs (1 hour to pull from S3)
This was with 1024*1024: 
https://github.visualstudio.com/electron/_build/results?buildId=2534&view=logs (13 minutes to pull from S3).

I believe AppVeyor was having the same issue, but strangely CircleCI was not.  Maybe because CircleCI uses AWS vms.

Reply:
Here's a fun question, how fast is it if you remove all the logging during download on CI?  Could we save another few minutes turning off these logs during CI builds?

Reply:
Maybe... I can check

Reply:
Removing the logging reduced the update step by 5 minutes.

I moved it to a verbose setting which is off by default.